lib/vagrant-service-manager/services/docker.rb

```diff
@@ -56,6 +58,10 @@ def self.info(machine, ui, options = {})
       end
 
       def self.print_env_info(ui, options)
+        if PluginUtil.debug_mode?
+          PluginUtil.log("script_readable: #{options[:script_readable] || false}", 'STDOUT')
```

Comment:
I can see the debug log are added using different ways.  e.g. 
* PluginUtil.log("script_readable: #{options[:script_readable] || false}", 'STDOUT'
* GuestUtil.log(options[:logger], "\"#{data.chop}\"", 'STDOUT') if options[:debug]

We need just one method to add debug logs.